CalendarApp/Models/EventModel.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  EventModel.swift
+//  CalendarApp
+//
+//  Created by Angelina Zhang on 7/18/22.
+//
+
+import Foundation
+import CalendarKit
+
+@objcMembers
+public class Event : NSObject {
+    public var parseObjectId: String?
+    public var ekEventID: String?
+    public var objectUUID = UUID()
+    public var updatedAt: Date?
+    public var createdAt: Date?
+    
+    public var eventTitle = ""
+    public var authorUsername: String?
+    public var eventDescription = ""
+    public var location = ""
+    
+    public var startDate = Date()
+    public var endDate = Date()
+    public var isAllDay = false
+    
+    public var color = SystemColors.systemBlue
+    public weak var editedEvent: EventDescriptor?
```

Comment:
This one likely doesn't need to be public if EventDescriptor isn't available to ObjC.